api/internal/handler/route_online_debug/route_online_debug.go

```diff
@@ -134,7 +132,20 @@ func (h *HTTPProtocolSupport) RequestForwarding(c droplet.Context) (interface{},
 
 	defer resp.Body.Close()
 
-	_body, err := ioutil.ReadAll(resp.Body)
+	// handle gzip content encoding
+	var reader io.ReadCloser
+	switch resp.Header.Get("Content-Encoding") {
```

Comment:
I have a question about this gzip header option:

when I try to request `http://httpbin.org/gzip`

I got the result like this:
![image](https://user-images.githubusercontent.com/2561857/115122037-64f5ee00-9fe8-11eb-8d31-1b408abf404b.png)

but I could not found `Accept-Encoding` in the response header tab content:
![image](https://user-images.githubusercontent.com/2561857/115122112-cc13a280-9fe8-11eb-932a-9725a0f3ec00.png)

why this header option missed?

Reply:
fixed. The reason is that the default golang http client configuration is to automatically remove the "Content-Encoding" header if "Accept-Encoding" is not found in the request.